lib/cog/queries/command.ex

```diff
@@ -43,6 +54,16 @@ defmodule Cog.Queries.Command do
   end
 
   @doc """
+  Retrieve all information about a command
+  """
+  def complete_command(bundle_name, command_name) do
+    named(bundle_name, command_name)
+    |> with_bundle
+    |> with_rules
+    |> with_options
```

Comment:
Ah you're right. I have gone a similar route as you have (adding composable functions for preloads), but given the number of combinations you end up needing to write functions for it got a little out of hand and I just opted to keep the preloads near where I end up using that data, which ends up being in a pipelined `Repo.all |> Repo.preload` call. I also like that you can easily combine all preloads into a single list when using `Repo.preload`, but this seems totally fine for now.